README.md

```diff
@@ -1 +1,156 @@
-# leaderboard-proj
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 To Do List Project ](#-to-do-list-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+      - [Live Demo Link](#live-demo-link)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Deployment](#deployment)
+  - [👥 Authors](#-authors)
+  - [👥 Authors](#-authors-1)
+  - [🔭 Future-feature ](#-future-feature-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Leaderboard Project <a name="about-project"></a>
+
+**Leaderboard ** allows users to submit and get name and scores to/from server
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Display scores**
+- **Submit scores**
+- **Preserve data**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+#### <a href="https://kamranahmad123.github.io/leaderboard-proj/dist/">live link</a>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Setup
+1- Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/kamranahmad123/leaderboard/tree/development
+```
+2- Run npm install for running the project locally after clonning up:
+
+3- Run 'npm start' for running the project:
+
```

Comment:
- [ ] It would be nice if you make your **Prerequisites**, **Setup**, **Install**, and **Usage** sub-sections something like the below to make your readme file descriptive and professional :blush:
- [ ] Also please don't forget to change your git clone command with my command below because your command is not working and giving a fatal error :fearful:
### Prerequisites
```
Visual Studio Code
Chrome Browser
Node.js
```
### Setup
```
cd my-folder
git clone git@github.com:kamranahmad123/leaderboard-proj.git
cd leaderboard-proj
```
### Install
```
npm install
```
### Usage
```
npm run build
npm start
```